hdl/jpegenc_v2/myhdl/rgb2ycbcr.py

```diff
@@ -0,0 +1,78 @@
+#!/bin/python
+from myhdl import *
+from random import randrange
+
+Y1, Y2, Y3 = 4899, 9617, 1868
+CB1, CB2, CB3 = -2764, -5428, 8192
+CR1, CR2, CR3 = 8192, -6860, -1332
+Y_OFFSET, CB_OFFSET, CR_OFFSET = 0, 2097152, 2097152
+
+def rgb2ycbcr(y, cb, cr, r, g, b, clk, reset):
+	""" A RGB to YCbCr converter with reset.
+
+	I/O pins:
+	--------
+	y       : output 8-bit unsigned value in range of 0-255
+	cb      : output 8-bit unsigned value in range of 0-255
+	cr      : output 8-bit unsigned value in range of 0-255
+	r 		: input 8-bit unsigned value in range of 0-255
+	g 		: input 8-bit unsigned value in range of 0-255
+	b 		: input 8-bit unsigned value in range of 0-255
+	clk 	: input clock boolean signal
+	reset	: input 8-bit boolean signal
+
+	"""
+
+	@always(clk.negedge, reset.posedge)
```

Comment:
I was thinking of using negedge in all modules. But it doesn't matter. Changing it to posedge.